src/main/java/org/jenkins/tools/test/hook/MultiParentCompileHook.java

```diff
@@ -37,57 +37,52 @@ public MultiParentCompileHook() {
     }
 
     @Override
-    @SuppressFBWarnings(value = "REC_CATCH_EXCEPTION", justification = "silly rule")
-    public Map<String, Object> action(Map<String, Object> moreInfo) throws Exception {
-        try {
-            LOGGER.log(Level.INFO, "Executing multi-parent compile hook");
-            PluginCompatTesterConfig config = (PluginCompatTesterConfig) moreInfo.get("config");
-
-            runner = new ExternalMavenRunner(config.getExternalMaven());
-            mavenConfig = getMavenConfig(config);
-
-            File pluginDir = (File) moreInfo.get("pluginDir");
-            LOGGER.log(Level.INFO, "Plugin dir is {0}", pluginDir);
-
-            File localCheckoutDir = config.getLocalCheckoutDir();
-            if (localCheckoutDir != null) {
-                Path pluginSourcesDir = localCheckoutDir.toPath();
-                boolean isMultipleLocalPlugins =
-                        config.getIncludePlugins() != null && config.getIncludePlugins().size() > 1;
-                // We are running for local changes, let's copy the .eslintrc file if we can. If we
-                // are using localCheckoutDir with multiple plugins the .eslintrc must be located at
-                // the top level. If not it must be located on the parent of the localCheckoutDir.
-                if (!isMultipleLocalPlugins) {
-                    pluginSourcesDir = pluginSourcesDir.getParent();
-                }
-                // Copy the file if it exists
-                try (Stream<Path> walk = Files.walk(pluginSourcesDir, 1)) {
-                    walk.filter(this::isEslintFile).forEach(eslintrc -> copy(eslintrc, pluginDir));
-                }
-            }
+    public Map<String, Object> action(Map<String, Object> moreInfo) throws PomExecutionException {
```

Comment:
Typical example of how to adapt a hook to these changes: remove the `throws Exception`, handle `IOException` internally (by rethrowing as `UncheckedIOException`, and then declare the method as throwing some checked exception if still necessary.